src/newrelic/infinite_tracing/trace_observer.go

```diff
@@ -112,7 +113,8 @@ type metricIncrement struct {
 type traceObserverSupportability struct {
 	incrementSent chan float64
 	increment     chan metricIncrement
-	dump          chan map[string]float64
+	dataUsage     chan float64
+	dump          chan map[string][6]float64
```

Comment:
Is there a special reason we want to use `6` here?

Reply:
Metrics are stored in the agent (and NR) as a [6]float. Previously, we were just creating "count" metrics in the trace_observer, and the interface we have in the daemon to handle that just took the single count (but then stored it in a [6]float). Now that data usage metrics are added, there are non-count metrics being created, which need multiple slots in the [6]float.

The options to address this are 1) create a new dump chan that is [6]float that only takes the data usage metrics and therefore use 2 channels for metrics on top of some other overhead or 2) (what I did) increase the width of the channel to account for data usage metrics coming through, even though most of the metrics will only be using 1 slot of the 6